packages/components/src/components/eventsCookieConsent/EventsCookieConsent.tsx

```diff
@@ -0,0 +1,114 @@
+import type { ContentSource } from 'hds-react';
+import { CookieModal } from 'hds-react';
+import React from 'react';
+import { MAIN_CONTENT_ID } from '../../constants';
+import { useConsentTranslation } from '../../hooks';
+import useLocale from '../../hooks/useLocale';
+import type { Language } from '../../types';
+
+type Props = {
+  appName: string;
+};
+
+const EventsCookieConsent: React.FC<Props> = ({ appName }) => {
+  const locale = useLocale();
+  const { t, i18n } = useConsentTranslation();
+  const [language, setLanguage] =
+    React.useState<ContentSource['currentLanguage']>(locale);
+
+  const onLanguageChange = React.useCallback(
+    (newLang: string) => {
+      setLanguage(newLang as Language);
+      i18n.changeLanguage(newLang);
```

Comment:
There is a bug in the language change. The language is not handled properly: only some parts of the UI changes.

This was discussed in some meeting we had, a retrospective or sprint planning or something, that the language change is not needed at all in the cookie consent. The application language change in the header should be the only language selector. 

- [ ] The language selector from the cookie consent should be removed

Reply:
However, since we are in hurry with this PR, we can do this change in another PR.